linkerd/app/inbound/src/policy/http/tests.rs

```diff
@@ -363,6 +375,110 @@ async fn http_filter_inject_failure() {
     );
 }
 
+#[tokio::test(flavor = "current_thread")]
+async fn rate_limit_allow() {
+    use linkerd_app_core::{Ipv4Net, Ipv6Net};
+
+    let rmeta = Meta::new_default("default");
+
+    // Rate-limit with plenty of room for two consecutive requests
+    let rl = LocalRateLimit::new_no_overrides_for_test(Some(10), Some(5));
+
+    let authorizations = Arc::new([Authorization {
+        meta: rmeta.clone(),
+        networks: vec![Ipv4Net::default().into(), Ipv6Net::default().into()],
+        authentication: Authentication::Unauthenticated,
+    }]);
+
+    let (mut svc, _tx) = new_svc!(
+        Protocol::Http1(Arc::new([http::default(authorizations.clone())])),
+        rl
+    );
+
+    // First request should be allowed
+    let rsp = svc
+        .call(
+            ::http::Request::builder()
+                .body(hyper::Body::default())
+                .unwrap(),
+        )
+        .await
+        .expect("serves");
+    let permit = rsp
+        .extensions()
+        .get::<HttpRoutePermit>()
+        .expect("permitted");
+    assert_eq!(permit.labels.route.route, rmeta);
```

Comment:
We can omit the checks that are covered by other tests--they're not necessary to test the rate limiting behavior.

```suggestion
    assert_eq!(rsp.status(), ::http::StatusCode::OK);
```